### ISSUE TITLE
there is no advice/hint/guide how this remote control feature works

### ISSUE THREAD

**Issue by lhpitn** [KRTirtho/spotube#1847]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

in the settings there is some ominous on of switch labeld something like "enable connection -control spotube from another device" (translated back from german) that i can switch on but i dont know what it dose, before i know what it dose i switched it back to the off postion but it is not how i like an intuitive ui

### Expected Behavior

some hint to the help me understand what interface is opened and how to use it, if it is just open for everybody or i have to whitelist devices on connect, also some port would be helpful in case i have a firewall

### Steps to reproduce

open the app, go to settings scroll down, 

### Operating System

Android 13

### Spotube version

v3.8.0 build 33 stable

### Installation source

GitHub Releases (Binary)

### Additional information

the ui is in general better looking then working, much testing has to be done on different display sizes and live devices with uptimes bigger then 7 days that run a lot of apps on that cupple of ram anandroid has ... sometimes i get some obscure sesion login error and have to kill the whole app using device settings menu (its a hazzle to find the app everytime in the list) to get it back to work, maybe just detect and relog souch states by your own using a little service health check/ control/restart thread...

### Self grab

- [X] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
Yeah I probably should write proper docs for it in the Wiki section.

But briefly, it's just a server running on `0.0.0.0:<random port>` which discoverable via avahi(zeroconf) as `_spotube.tcp` type. It also shares 2 attributes `id` and `deviceId` 

The app uses [`WebSockets`](https://en.wikipedia.org/wiki/WebSocket) at `ws://0.0.0.0:port/ws` endpoint to communicate bidirectionally. 

You can see the events and actions at:
https://github.com/krtirtho/spotube/blob/86ee64c6066b3aa3c5910c89ffff7367826e823c/lib/provider/server/routes/connect.dart
